ipamutils/utils_test.go

```diff
@@ -24,3 +27,24 @@ func TestGranularPredefined(t *testing.T) {
 	}
 
 }
+
+func TestInitAddressPools(t *testing.T) {
+	prePool1 := &PredefinedPools{"172.80.0.0/16", 24}
+	prePool2 := &PredefinedPools{"172.90.0.0/16", 24}
+	DefaultAddressPools := []*PredefinedPools{prePool1, prePool2}
+	initNetworksOnce = sync.Once{}
+	InitNetworks(DefaultAddressPools)
+
+	// Check for Random IPAddresses in PredefinedBroadNetworks  ex: first , last and middle
+	if len(PredefinedBroadNetworks) == 512 {
```

Comment:
instead of the if, else, use `assert.Len`

Reply:
done